ew/static/cosmetics.py

```diff
@@ -1927,7 +1927,7 @@
     EwCosmeticItem(
         id_cosmetic = "yarmulke",
         str_name = "Yarmulke",
-        str_desc = "Oy Vey!", # For the love of all that is holy please do not smite me down, Igonis was the one behind this item.
+        str_desc = "Barukh Attah Adonai. May you keep your faith in your lord; the one true god, Yahweh", # Igonis is gone, I'm not keeping Oy Vey as the description text as ignois's request.
```

Comment:
why is igonis's flavor text verboten all of a sudden? or are you saying he asked it be changed? seems innocuous enough